funktionale-try/src/test/kotlin/org/funktionale/tries/TryTest.kt

```diff
@@ -59,6 +59,11 @@ class TryTest {
         failure.foreach { fail() }
     }
 
+    @Test fun onEach() {
```

Comment:
Sorry for the late response.

Personally, I don't mind either way as long as the commits make sense. I'm not sure what's Mario's opinion on it though.